src/main/mdo/dependency-updates-report.mdo

```diff
@@ -21,8 +21,8 @@ under the License.
 
 <model xmlns="http://codehaus-plexus.github.io/MODELLO/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://codehaus-plexus.github.io/MODELLO/2.0.0 https://codehaus-plexus.github.io/modello/xsd/modello-2.0.0.xsd"
-       xsd.namespace="http://mojo.codehaus.org/versions-maven-plugin/dependency-updates-report/1.0.0"
-       xsd.targetNamespace="http://mojo.codehaus.org/versions-maven-plugin/dependency-updates-report/1.0.0">
+       xml.namespace="https://www.mojohaus.org/VERSIONS/DEPENDENCY-UPDATES-REPORT/${version}"
```

Comment:
Do we really want to change the namespace of the reports with every new release of the plugin?

Reply:
It is `version` of model form `modello-maven-plugin` configuration  - it is not project version